strip-coverletter.sh

```diff
@@ -20,6 +20,7 @@ errcho() { echo "$@" 1>&2; } # write to stderr
 
 infile=$1
 outfile=$2
+squash=${3:-"--squash=false"}
```

Comment:
I think a `--` option creates a false sense of security in passing it in any position, while it would only work as the 3rd argument. I'd strip it down to a single value like `true`|`false`, which is less readable on the calling side but makes the interface foolproof.

Reply:
I had just true/false originally but it looked and felt wrong so I changed it to the above. I do agree with you though. Even something like `squash` or `nosquash` as the third parameter without the `--` still feels like a positional. I'll revert it to simply true/false